libindy-crypto/src/bn/openssl.rs

```diff
@@ -2,8 +2,8 @@ use errors::IndyCryptoError;
 
 use int_traits::IntTraits;
 
-use openssl::bn::{BigNum, BigNumRef, BigNumContext, MSB_MAYBE_ZERO};
-use openssl::hash::{hash2, MessageDigest, Hasher};
+use openssl::bn::{BigNum, BigNumRef, BigNumContext, MsbOption};
+use openssl::hash::{hash, MessageDigest, Hasher};
```

Comment:
I hope it's correct that `hash` corresponds to `hash2` in the old version of openssl. `hash2` is no more, leaving only `hash` in the current API, and I don't know the definition of `hash2`.